addons/test_mail_full/tests/test_sms_performance.py

```diff
@@ -51,7 +51,7 @@ def setUp(self):
     def test_message_sms_record_1_partner(self):
         record = self.test_record.with_user(self.env.user)
         pids = self.customer.ids
-        with self.mockSMSGateway(sms_allow_unlink=True), self.assertQueryCount(employee=23):  # test_mail_enterprise: 25
+        with self.mockSMSGateway(sms_allow_unlink=True), self.assertQueryCount(employee=24):  # test_mail_enterprise: 25
```

Comment:
Did you check for the value in comment for `test_mail_enterprise`? Maybe it should be updated too just for clarity.